docs/configuration.rst

```diff
@@ -259,6 +259,20 @@ Panel options
   This setting affects the depth of function calls in the profiler's
   analysis.
 
+* ``PROFILER_THRESHOLD_RATIO``
+
+  Default: ``10``
```

Comment:
```suggestion
  Default: ``8``
```

I have no idea what this value even means :-) but the code says that the default is 8.